code/modules/mob/living/simple_mob/subtypes/vore/corrupt_hounds.dm

```diff
@@ -87,40 +87,40 @@
 		..()
 
 
-/mob/living/simple_mob/vore/corrupthound/add_eyes()
+/mob/living/simple_mob/vore/aggresive/corrupthound/add_eyes()
 	if(!eye_layer)
 		eye_layer = image(icon, "badboi-eyes")
 		eye_layer.plane = PLANE_LIGHTING_ABOVE
 	add_overlay(eye_layer)
 
-/mob/living/simple_mob/vore/corrupthound/remove_eyes()
+/mob/living/simple_mob/vore/aggresive/corrupthound/remove_eyes()
 	cut_overlay(eye_layer)
 
-/mob/living/simple_mob/vore/corrupthound/New()
+/mob/living/simple_mob/vore/aggresive/corrupthound/New()
 	add_eyes()
 	..()
 
-/mob/living/simple_mob/vore/corrupthound/death(gibbed, deathmessage = "shudders and collapses!")
+/mob/living/simple_mob/vore/aggresive/corrupthound/death(gibbed, deathmessage = "shudders and collapses!")
 	.=..()
 	resting = 0
 	icon_state = icon_dead
 
-/mob/living/simple_mob/vore/corrupthound/update_icon()
+/mob/living/simple_mob/vore/aggresive/corrupthound/update_icon()
 	. = ..()
 	remove_eyes()
 	if(stat == CONSCIOUS && !resting)
 		add_eyes()
 
 /* //VOREStation AI Temporary Removal
-/mob/living/simple_mob/vore/corrupthound/Login()
+/mob/living/simple_mob/vore/aggresive/corrupthound/Login()
```

Comment:
Not sure if ridingcode got fixed, might be possible to readd these lines, also aggressive

Reply:
out of scope